website/content/en/preview/upgrading/v1-migration.md

```diff
@@ -0,0 +1,247 @@
+---
+title: "v1 Migration"
+linkTitle: "v1 Migration"
+weight: 30
+description: >
+  Upgrade information for migrating to v1
+---
+
+This migration guide is designed to help you migrate Karpenter from a pre-v1 version (v1beta1) to Karpenter v1.
+Use this document as a reference to the changes that were introduced in this release and as a guide to how you need to update the manifests and other Karpenter objects you created in previous Karpenter releases.
+
+Before you begin upgrading to `1.0.0`, you should know that:
+
+* Every Karpenter upgrade from pre-v1.0.0 versions must go through an upgrade to minor version `v1.0.0`.
+* You must be upgrading to `v1.0.0` from a version of Karpenter that supports NodePools, NodeClaims, and NodeClasses (`0.32.0`+ Karpenter versions support v1beta1 APIs).
+* Karpenter `1.0.0`+ supports Karpenter v1 and v1beta1 APIs and will not work with earlier Provisioner, AWSNodeTemplate or Machine alpha APIs. Do not upgrade to `1.0.0`+ without first [upgrading to `0.32.x`]({{<ref "upgrade-guide#upgrading-to-0320" >}}) or later.
+* Because version `1.0.0` supports both the v1beta1 and v1 APIs, it allows you to migrate your existing APIs without experiencing downtime.
+
+## Upgrade Procedure
+
+This procedure assumes you are running the Karpenter controller on cluster and want to upgrade that cluster to v1.0.0.
+
+**NOTE**: Please read through the entire procedure before beginning the upgrade. There are major changes in this upgrade, so you should carefully evaluate your cluster and workloads before proceeding.
+
+1. Determine the current cluster version: Run the following to make sure that your Karpenter version is between `v0.32.11` and `v0.37.1`:
+   ```bash
+   kubectl get pod -A | grep karpenter
+   kubectl describe pod -n karpenter karpenter-xxxxxxxxxx-xxxxx | grep Image: 
+   ```
+   Sample output:
+   ```bash
+   Image: public.ecr.aws/karpenter/controller:0.37.1@sha256:157f478f5db1fe999f5e2d27badcc742bf51cc470508b3cebe78224d0947674f
+   ```
+
+   The Karpenter version you are running must be between minor version `v0.32` and `v0.37`. To be able to roll back from Karpenter v1, we recommend that you also be on at least the following patch release versions for your minor version:
+
+   * v0.37.1
+   * v0.36.3
+   * v0.35.6
+   * v0.34.7
+   * v0.33.6
+   * v0.32.11
```

Comment:
```suggestion
```

Users are expected to be on v0.33.x or above

Reply:
Done.